sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -2298,4 +2298,15 @@ public void testComplexCoderSerializable() {
     }
     return converted;
   }
+
+  @Test
```

Comment:
Is it possible to add a test more representative of the user's failing pipeline?